google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/it/BuiltinMetricsIT.java

```diff
@@ -58,64 +88,174 @@ public class BuiltinMetricsIT {
+            .setIsolationPolicy(AppProfile.StandardIsolationPolicy.of(AppProfile.Priority.LOW)));
+
+    metricReader = InMemoryMetricReader.create();
+
+    SdkMeterProviderBuilder meterProvider =
+        SdkMeterProvider.builder().registerMetricReader(metricReader);
+    BuiltinMetricsView.registerBuiltinMetrics(testEnvRule.env().getProjectId(), meterProvider);
+    OpenTelemetry openTelemetry =
+        OpenTelemetrySdk.builder().setMeterProvider(meterProvider.build()).build();
+
+    BigtableDataSettings.Builder settings = testEnvRule.env().getDataClientSettings().toBuilder();
+
+    clientCustomOtel =
+        BigtableDataClient.create(
+            settings
+                .setMetricsProvider(CustomOpenTelemetryMetricsProvider.create(openTelemetry))
+                .setAppProfileId(appProfileCustomOtel)
+                .build());
+    clientDefault = BigtableDataClient.create(settings.setAppProfileId(appProfileDefault).build());
   }
 
-  @AfterClass
-  public static void tearDown() {
+  @After
+  public void tearDown() {
     if (metricClient != null) {
       metricClient.close();
     }
-    if (table != null) {
-      tableAdminClient.deleteTable(table.getId());
+    if (tableCustomOtel != null) {
+      tableAdminClient.deleteTable(tableCustomOtel.getId());
+    }
+    if (tableDefault != null) {
+      tableAdminClient.deleteTable(tableDefault.getId());
+    }
+    if (instanceAdminClient != null) {
+      instanceAdminClient.deleteAppProfile(
+          testEnvRule.env().getInstanceId(), appProfileCustomOtel, true);
+      instanceAdminClient.deleteAppProfile(
+          testEnvRule.env().getInstanceId(), appProfileDefault, true);
+    }
+    if (clientCustomOtel != null) {
+      clientCustomOtel.close();
+    }
+    if (clientDefault != null) {
+      clientDefault.close();
+    }
+  }
+
+  @Test
+  public void testBuiltinMetricsWithDefaultOTEL() throws Exception {
+    logger.info("Started testing builtin metrics with default OTEL");
+    tableDefault =
+        tableAdminClient.createTable(
+            CreateTableRequest.of(PrefixGenerator.newPrefix("BuiltinMetricsIT#test1"))
+                .addFamily("cf"));
+    logger.info("Create default table: " + tableDefault.getId());
+    clientDefault.mutateRow(
+        RowMutation.create(tableDefault.getId(), "a-new-key").setCell("cf", "q", "abc"));
+    ArrayList<Row> rows =
+        Lists.newArrayList(clientDefault.readRows(Query.create(tableDefault.getId()).limit(10)));
+
+    Stopwatch stopwatch = Stopwatch.createStarted();
+
+    ProjectName name = ProjectName.of(testEnvRule.env().getProjectId());
+
+    Collection<MetricData> fromMetricReader = metricReader.collectAllMetrics();
+
+    // Restrict time to last 10 minutes and 5 minutes after the request
+    long startMillis = System.currentTimeMillis() - Duration.ofMinutes(10).toMillis();
+    long endMillis = startMillis + Duration.ofMinutes(15).toMillis();
```

Comment:
This kinda defeats the purpose of Duration. In general you want to keep your measurements wrapped in duration until the last moment to reduce the chance of mixing up units:

```
Instant start = Instant.now().minus(Duration.ofMinutes(10));
Instant end = start.plus(Duration.ofMinutes(15);

...
setStartTime(start.toEpochMilli())
```

Reply:
Im also confused about this time window why so big?